docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+
+
+### What does success look like for the EVM sidechain after launch on XRPL Mainnet?
+
+Long term, the EVM sidechain has a large and sustainable ecosystem of developers. The XRPL developer ecosystem has adopted the EVM sidechain as an extension of the XRPL and uses it as a testing ground before features are proposed on XRPL Mainnet or has deployed apps there that are not suitable for the XRPL Mainnet.
+
+
+### Who is our target user for the EVM sidechain?
+
+An EVM application developer who has experience writing Solidity applications and who is looking to monetize their applications on other EVM compatible blockchains outside of Ethereum. This developer isn’t happy about the high gas fees and low transaction speeds on Ethereum during peak usage.
+
+Another segment of EVM developers is the EVM core developer that works on new protocol primitives on Ethereum or similar blockchains. They have less opportunities to contribute to Ethereum so the EVM sidechain provides them a blank slate to add new innovative capabilities.
+
+
+### What consensus algorithm does the EVM sidechain use?
+
+The EVM sidechain will use the Proof of Authority (PoA) consensus algorithm. A set of validators will confirm blocks on the EVM sidechain. New validators can join, existing validators can leave or based on voting by the existing set of validators.
+
+
+### Why should an EVM developer use the EVM sidechain of the XRPL as opposed to Ethereum or any other EVM compatible blockchain?
+
+An EVM developer that builds on the EVM sidechain essentially gets a blank slate to start from. There will be considerable excitement to deploy the first apps that were successful on other EVM blockchains on the EVM sidechain. The EVM sidechain has a 3.5 second block time  which is much lower than the 12-15 seconds used on Ethereum. XRP is one of the most liquid cryptocurrencies, and developers can reach millions of XRP holders by launching dapps on the EVM sidechain where XRP is a native gas token.
+
+
+### Are there any incentives for EVM sidechain validator nodes?
+
+The EVM sidechain will not have any block rewards. The transaction fee for transactions confirmed on the EVM sidechain will be distributed to EVM sidechain validator nodes. This is unlike the XRPL where this transaction fee is burnt.
+
+
+### What applications are recommended to be developed on XRPL vs EVM sidechain?
+
+Developers are free to choose any blockchain they wish. The XRPL has a robust set of native primitives that are secure, performant and with low transaction cost. Developers can build using these primitives natively on the XRPL.
+
+On the other hand, the XRPL has limited programmability options for developers to change business logic in the primitives that exist. The ability to readily deploy programmable applications, combined with ample XRP liquidity from XRPL Mainnet and 3.5 second block times, makes the EVM sidechain especially optimized for DeFi use cases.
```

Comment:
The EVM sidechain is especially optimized for decentralized finance use cases, with its readily deployable programmable applications, ample XRP liquidity from Mainnet, and 3.5 second block times.